.github/workflows/main.yml

```
@@ -1,7 +1,7 @@
 name: main
 on: 
   workflow_run:
-    workflows: [dev]
+    workflows: [dev, prod]
```

Is this supposed to use the new workflow's name?
```suggestion
    workflows: [dev, deploy]
```

---

`main` reports a failure on the main or production branches to slack since we'll want to resolve any issue here ASAP. 

I omitted `deploy`, but it could be included here as well.